frontend/src/components/editor/Cell.tsx

```diff
@@ -104,6 +109,7 @@ export interface CellProps
    */
   allowFocus: boolean;
   userConfig: UserConfig;
+  isScratchpad?: boolean;
```

Comment:
i think rather than using this `<Cell` for the scratchpad - maybe we can re-use different smaller components like `<OutputArea/>`

i am happy to take on splitting this up though if you prefer